assume/markets/base_market.py

```diff
@@ -397,7 +397,10 @@ async def clear_market(self, market_products: list[MarketProduct]):
 
         for meta in market_meta:
             logger.debug(
-                f'clearing price for {self.marketconfig.name} is {meta["price"]:.2f}, volume: {meta["demand_volume"]}'
+                "clearing price for %s is %.2f, volume: %f",
+                self.marketconfig.name,
```

Comment:
can we please stick to one format for strings? In the whole code we are using f"{}" format. it would be confusing to have it different in some places.

Reply:
I had this view too for quite a while.
But unfortunately, formatting with f-strings is evaluated every time - even if the logger is set to warning (so the price is formatted even if the line is not printed).
Using it with the % notation does only execute the formatting when loglevel is set to debug.

But I would not like to switch to the %s notation on all other places.. :shrug:
So I think we can't really do anything but to use the %s format in logger situations and format strings everywhere else

Reply:
hmm, good to know, let's keep it this way